roles/lib_utils/library/docker_creds.py

```diff
@@ -200,6 +206,7 @@ def run_module():
         password=dict(type='str', required=True, no_log=True),
         test_login=dict(type='bool', required=False, default=True),
         proxy_vars=dict(type='str', required=False, default=''),
+        test_timeout=dict(type='str', required=False, default='20'),
```

Comment:
```suggestion
        test_timeout=dict(type='int', required=False, default='20'),
```

Reply:
@mtnbikenc Thanks for catching that. Fixed